linkerd/dns/name/src/name.rs

```diff
@@ -12,7 +12,7 @@ pub struct InvalidName;
 
 impl Name {
     pub fn is_localhost(&self) -> bool {
-        *self == Name::try_from("localhost.".as_bytes()).unwrap()
+        *self == Name::try_from(&b"localhost."[..]).unwrap()
```

Comment:
could be `from_str` now

Reply:
yeah --- the clippy lint was specifically about using byte string literals rather than `.as_bytes()`.